pkgs/applications/video/obs-studio/plugins/obs-browser-transition.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  obs-studio,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
consider switching to finalAttrs

Reply:
Pushed an update to address the feedback.